.github/workflows/deploy.yaml

```diff
@@ -29,3 +29,4 @@ jobs:
       - run: yarn subgraph deploy ${{ matrix.chain }}
         env:
           TOKEN: ${{ secrets.THEGRAPH_TOKEN }}
+          VERSION: ${{ github.event.release.tag_name }}
```

Comment:
If you want to eliminate the versioning process, which can be painful sometimes, you can keep the subgraph deployment on push events and use the commit hash (or a shortened rendering) as the version number.

If needed, here's an example of an action to obtain the shortened hash to place in `.github/actions/short-sha/action.yml`:

```yaml
name: short-sha
description: Compute release version
runs:
  using: composite
  steps:
    - id: short-sha
      shell: sh
      run: echo "SHORT_SHA=`echo ${{ github.sha }} | cut -c 1-7`" >> $GITHUB_OUTPUT
outputs:
  hash:
    description: The short Git HEAD hash
    value: ${{ steps.short-sha.outputs.SHORT_SHA }}
```

It can be used like this:
```yaml
steps:
  - uses: ./.github/actions/short-sha
    id: short-sha

  ...

  VERSION: ${{ steps.short-sha.outputs.hash }}
```

I'll let you decide what you prefer 🙂 Both methods have pros and cons

Reply:
I'm leaning toward using the short-sha as a version number. What do you think @skiv71?